include/triton/Conversion/TritonGPUToLLVM/PatternTritonGPUOpToLLVM.h

```diff
@@ -27,15 +27,32 @@ constexpr int patternBenefitPrioritizeOverLLVMConversions = 10;
 constexpr int patternBenefitClampOptimizedPattern = 20;
 constexpr int patternBenefitConvertLayoutOptimizedPattern = 20;
 
+struct BackendCallbacks {
+  /**
+   * The reference to a backend-specific callback for appending auxiliary data
```

Comment:
Nit: "A backend-specific callback for .."

Reply:
Done